_posts/2019-09-09-what-is-mtls.md

```diff
@@ -49,7 +49,9 @@ But how does it actually work? We've mentioned already the 'handshake' as a conc
 
 {% asset blog/tls-structure.png srcset:width="1300 2x" srcset:width="650 1x" alt="Breakdown of the TLS structure" %}
 
-The protocol/handshake layer is where the cryptographic 'magic' takes place. It manages the encryption algorithm used (also known as cipher) and key exchange, usually known as PKI (public key infrastructure). The PKI is signed for by the Certificate Authority and is what gives an endpoint its cryptographic identity. Basically, an x509 certificate is generated by this handshake protocol based off of the technologies each endpoint is using and then these certificates verified by the appropriate CA. With mTLS, it's the exact same except both the server and client verify each other. You can think of the CA as a signiature vouching for a device's identity.
+The protocol/handshake layer is where the cryptographic 'magic' takes place. It manages the encryption algorithm used (also known as cipher) and key exchange, usually known as PKI (public key infrastructure). The PKI is signed for by the Certificate Authority and is what gives an endpoint its cryptographic identity. Basically, the server being accessed has an associated x509 certificate from a CA which is exchanged in this handshake protocol. This certificate contains information regarding the server's name, its public key (PKI), and a 'signiature.' This is why we often refer to this as 'digtal signage.' 
```

Comment:
> usually known as PKI (public key infrastructure)

No, the PKI is something different. The PKI is the system for obtaining the certificate. I.e. the server that will provide you with the actual certificate (which may include multiple servers) as well as revocation lists etc. See [PKI](https://en.wikipedia.org/wiki/Public_key_infrastructure) on Wikipedia for more details. Read over that page and let me know if that if you have any questions.